Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -180,3 +180,29 @@ def customiseRun2CommonHI(process):
 
     return process
 
+##############################################################################
+# modifications of muon DQMOffline for pPb run
+def customiseRun2PPB_MuonDQMOffline(process):
```

Comment:
Hi @slava77 , I just want to ask you before update it. 
So you mean we will use DQMOfflineHeavyIons_cff.py during pPb Run?